storage/innobase/include/btr0scrub.h

```diff
@@ -28,13 +28,12 @@ enum btr_scrub_page_allocation_status_t {
 #define BTR_SCRUB_SKIP_PAGE_AND_CLOSE_TABLE 3    /* no scrub & close table */
 #define BTR_SCRUB_SKIP_PAGE_AND_COMPLETE_SPACE 4 /* no scrub & complete space \
                                                   */
-#define BTR_SCRUB_TURNED_OFF      \
-  5 /* we detected that scrubbing \
-    was disabled by global        \
-    variable */
+#define BTR_SCRUB_TURNED_OFF                                      \
+  5 /* we detected that scrubbing                                 \
```

Comment:
Needs manual help: join the comment lines and then reformat

Reply:
There's something bugged here: repeated clang format calls keep changing the formatting... looks like it stabilizes after a few calls, I'll commit that version.